frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -0,0 +1,56 @@
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens 
+- recieve confirmation form the curation team will accept your dataset along with instructions on how to format your dataset for submission.
+- Prepare your dataset and email our curators for further assistance in the process
+- Publish!
+
+
+
+## Dataset Requirements
+
+We need the following collection metadata (i.e. details associate with your publication or study)
+
+- Collection information:
+  - Title
+  - Description
+  - Contact: name and email
+  - Publication/preprint DOI: can be added later
+  - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
+ 
+Datasets need the following information added to a single h5ad (AnnData) format file per cellxgene visualization:
+- **Dataset-level metadata in uns**:
+  - schema_version: '2.0.0'
+  - title: title of the individual dataset visualization
+  - X_normalization: method used to normalize the data stored in X and any scaling or logging; 'none' if data in X is raw 
+  - optional: batch_condition: list of obs fields that define that “batches” that a normalization or integration algorithm should be aware of
+- **Data in .X and raw.X**:
+  - raw counts are required
+  - normalized counts are strongly recommended
+  - raw counts should be in raw.X if normalized counts are in .X
+  - if there is no normalized matrix, raw counts should be in .X
+- **Cells metadata in obs (the most specific term available from the specified ontology)**:
+  - organism_ontology_term_id: NCBITaxon (NCBITaxon:9606 for human, NCBITaxon:10090 for mouse)
+  - tissue_ontology_term_id: UBERON
+  - assay_ontology_term _id: EFO
+  - disease_ontology_term _id: MONDO or PATO:0000461 for ‘normal’
+  - cell_type_ontology_term _id: CL
+  - ethnicity_ontology_term _id: HANCESTRO if human, ‘unknown’ if info unavailable, ‘na’ if non-human
+  - development_stage_ontology_term _id: HsapDv if human, MmusDv if mouse, ‘unknown’ if info unavailable
+  - sex_ontology_term_id: PATO:0000384 for male, PATO:0000383 for female, or ‘unknown’ if unavailable
+- **Embeddings in obsm**:
+  - One or more two-dimensional embeddings, prefixed with 'X_'
+- **Features in var & raw.var (if present)**:
+  - index is Ensembl ID
+  - preference is that gene have not been filtered in order to maximize future data integration efforts
+
+
```

Comment:
This is missing the data submission policy?

Reply:
+1

Reply:
_direct personal identifiers_ is a link to the information in - https://github.com/chanzuckerberg/single-cell-data-portal/blob/41af03dd63690210a245c00c05b38989a1769207/frontend/src/components/Collections/components/PublishCollection/components/Policy/components/DirectPersonalIdentifiers/index.tsx

Reply:
> direct personal identifiers is a link to the information in - https://github.com/chanzuckerberg/single-cell-data-portal/blob/41af03dd63690210a245c00c05b38989a1769207/frontend/src/components/Collections/components/PublishCollection/components/Policy/components/DirectPersonalIdentifiers/index.tsx

Added in 5a46c3da